src/main/java/FileUtils.java

```diff
@@ -0,0 +1,33 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
+public class FileUtils {
+
+    static Employee[] read(File fileName) throws FileNotFoundException {
+        Scanner scanner = new Scanner(fileName);
```

Comment:
Pamiętaj o zamykaniu pliku, albo najlepiej wprowadź try-with-resources https://youtu.be/LRwSHA9YDgY